src/components/Observations/ObsStatus.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import classnames from "classnames";
+import checkCamelAndSnakeCase from "components/ObsDetails/helpers/checkCamelAndSnakeCase";
+import {
+  ActivityCount,
+  QualityGradeStatus
+} from "components/SharedComponents";
+import { View } from "components/styledComponents";
+import type { Node } from "react";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { useTheme } from "react-native-paper";
+import Observation from "realmModels/Observation";
+
+type Props = {
+  item: typeof Observation,
```

Comment:
Is this correct @kueda ? Couldn't find any usage of specific types in the repo

Reply:
That's great if flow doesn't complain about it. However, I wanted to change the *name* of the prop to `observation` so it's clear to developers what it is, not just IDEs and linters.

Reply:
Ohhh, I see. Will do